base/pkg/resolve/maxsum.jl

```diff
@@ -122,7 +122,7 @@ mutable struct Graph
                         adjdict[p0][p1] = j1
 
                         bm = trues(spp[p1], spp[p0])
-                        bmt = adjoint(bm)
+                        bmt = trues(spp[p0], spp[p1])
```

Comment:
Wow, your re-write here is _way_ better.  Bravo for not just zoning out (like I did when this change caught me off guard).

Reply:
One small step towards atonement for the horde of ghastly transformations I introduced in #24969 😉.